src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchField {
+  display: block;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 68px;
+  margin: 20px 0;
+}
+
+.inputOfBigField {
+  position: absolute;
+  border: none;
+  outline: none;
+  margin-top: 23px;
+  padding: 0;
+  height: 22px;
+  font-family: Avenir, sans-serif ;
```

Comment:
move to `html` selector and use `inherit` where needed

Reply:
If you have questions, ask in fe_chat. we don't receive notifications from GH